data/config-schema/src/main/resources/net/corda/schema/configuration/reconcliation/1.0/corda.reconciliation.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I thought we had added this file with [#343](https://github.com/corda/corda-api/pull/343).